modules/social_features/social_featured_content/config/update/social_featured_content_update_10200.yml

```diff
@@ -0,0 +1,15 @@
+field.field.paragraph.featured.field_featured_description:
```

Comment:
Since we add this to 10.3 milestone, need to rename the number to start with 103

Reply:
fixed